model/weapon.html

```diff
@@ -1,133 +1,66 @@
 <form class="{{cssClass}}" autocomplete="off">
-    <div class="header">
+    <div class="header border">
         <div class="avatar">
             <img class="profile-img" src="{{item.img}}" data-edit="img" title="{{item.name}}" />
         </div>
-        <div class="name">
+        <div class="header-stats">
             <label>{{localize "GEAR.NAME"}}</label>
             <input name="name" type="text" value="{{item.name}}" />
-        </div>
-        <div class="weight">
-            <label>{{localize "GEAR.WEIGHT"}}</label>
-            <select name="data.weight">
-                {{#select data.weight}}
-                <option value="tiny">{{localize "WEIGHT.TINY"}}</option>
-                <option value="light">{{localize "WEIGHT.LIGHT"}}</option>
-                <option value="regular">{{localize "WEIGHT.REGULAR"}}</option>
-                <option value="heavy">{{localize "WEIGHT.HEAVY"}}</option>
-                {{/select}}
-            </select>
-        </div>
-        <div class="bonus">
+
             <label>{{localize "WEAPON.BONUS"}}</label>
-            <div class="multivalue">
-                <input name="data.bonus.value" type="number" value="{{data.bonus.value}}" data-dtype="Number" />
-                &nbsp;/&nbsp;
-                <input name="data.bonus.max" type="number" value="{{data.bonus.max}}" data-dtype="Number" />
+            <div class="bonus">
+                <div class="skulls {{#unless data.bonus.value }}broken{{/unless}}">
+                    <a class="change-bonus" data-attribute="{{data.bonus}}">
+                        {{#skulls data.bonus.value data.bonus.max}}
+                        <i class="skull far {{#if @damaged}}fa-times-circle{{else}}fa-circle{{/if}}"></i>
+                        {{/skulls}}
+                    </a>
+                </div>
+                <div class="multivalue">
+                    <input name="data.bonus.value" type="number" value="{{data.bonus.value}}" data-dtype="Number" />
+                    &nbsp;/&nbsp;
+                    <input name="data.bonus.max" type="number" value="{{data.bonus.max}}" data-dtype="Number" />
+                </div>
             </div>
-        </div>
-    </div>
-    <div class="item">
```

Comment:
There are some CSS rules that rely on this class being in the path. (E.g. the roll-modifiers are missing some styling because of this.) The easiest way to avoid the styles breaking is probably to re-introduce this class somewhere in the reworked HTML. (I see you do it for the artifact tab, but not on the supply tab.)